test/e2e/performance/data/macos/data.js

```diff
@@ -1 +1 @@
.5703000000000005},{"run":5,"LayoutDuration":0.16427199999999997,"RecalcStyleDuration":0.23827600000000015,"ScriptDuration":0.87751,"TaskDuration":1.4108350000000005}],"load":[{"run":1,"LayoutDuration":0.042195,"RecalcStyleDuration":0.035147000000000005,"ScriptDuration":0.021971,"TaskDuration":0.405554},{"run":2,"LayoutDuration":0.023354999999999994,"RecalcStyleDuration":0.035035,"ScriptDuration":0.010188,"TaskDuration":0.339586},{"run":3,"LayoutDuration":0.022187,"RecalcStyleDuration":0.032875,"ScriptDuration":0.010924000000000003,"TaskDuration":0.33594900000000005},{"run":4,"LayoutDuration":0.02291,"RecalcStyleDuration":0.033782999999999994,"ScriptDuration":0.010605999999999997,"TaskDuration":0.33426},{"run":5,"LayoutDuration":0.021916999999999992,"RecalcStyleDuration":0.032103999999999994,"ScriptDuration":0.010824,"TaskDuration":0.3352599999999999},{"run":1,"LayoutDuration":0.037064,"RecalcStyleDuration":0.051883,"ScriptDuration":0.024612,"TaskDuration":0.459436},{"run":2,"LayoutDuration":0.027950000000000003,"RecalcStyleDuration":0.040992,"ScriptDuration":0.010793,"TaskDuration":0.376775},{"run":3,"LayoutDuration":0.029176999999999995,"RecalcStyleDuration":0.04217,"ScriptDuration":0.012958000000000004,"TaskDuration":0.39595499999999995},{"run":4,"LayoutDuration":0.027638999999999997,"RecalcStyleDuration":0.04050200000000001,"ScriptDuration":0.011103999999999996,"TaskDuration":0.37871999999999995},{"run":5,"LayoutDuration":0.027963000000000016,"RecalcStyleDuration":0.040523,"ScriptDuration":0.013883,"TaskDuration":0.40293800000000024},{"run":1,"LayoutDuration":0.033171,"RecalcStyleDuration":0.04804,"ScriptDuration":0.023202,"TaskDuration":0.423848},{"run":2,"LayoutDuration":0.022262999999999998,"RecalcStyleDuration":0.033665,"ScriptDuration":0.010945999999999997,"TaskDuration":0.330909},{"run":3,"LayoutDuration":0.023983000000000004,"RecalcStyleDuration":0.035865999999999995,"ScriptDuration":0.010356000000000004,"TaskDuration":0.34630399999999995},{"run":4,"LayoutDuration":0.021722999999999992,"RecalcStyleDuration":0.03275499999999999,"ScriptDuration":0.011036999999999998,"TaskDuration":0.3456109999999999},{"run":5,"LayoutDuration":0.02229500000000001,"RecalcStyleDuration":0.0335,"ScriptDuration":0.010174999999999997,"TaskDuration":0.32194600000000007},{"run":1,"LayoutDuration":0.029065,"RecalcStyleDuration":0.041504,"ScriptDuration":0.021812,"TaskDuration":0.369482},{"run":2,"LayoutDuration":0.021920999999999996,"RecalcStyleDuration":0.032736,"ScriptDuration":0.009968999999999995,"TaskDuration":0.33135000000000003},{"run":3,"LayoutDuration":0.024193,"RecalcStyleDuration":0.035473000000000005,"ScriptDuration":0.009803000000000006,"TaskDuration":0.337842},{"run":4,"LayoutDuration":0.024107000000000003,"RecalcStyleDuration":0.035344,"ScriptDuration":0.010203999999999998,"TaskDuration":0.34043899999999994},{"run":5,"LayoutDuration":0.023278000000000007,"RecalcStyleDuration":0.034913,"ScriptDuration":0.010440999999999999,"TaskDuration":0.333164}]}
```

Comment:
It looks like there is more than 1 set of 5 runs added here
we supposed to add just one set per feature to not let the data files to BIG in the future

Reply:
Maybe the cleaning of this file is missing before adding the information on

Reply:
in fact, data is saved every time tests are launched - we can cover cleaning in #862

Reply:
I suggest to do the clean right now otherwise this will be a mess and we won't know what are the various set of runs. IMHO, doing it in #862 will be too late.
So I suggest that macOS data to be reset to the state prior the implementation of this PR and the dataset recreated.